src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php

```diff
@@ -33,6 +34,22 @@ public function getConfigTreeBuilder(): TreeBuilder
 
         $rootNode
             ->docUrl('https://symfony.com/doc/{version:major}.{version:minor}/reference/configuration/twig.html', 'symfony/twig-bundle')
+            ->beforeNormalization()
+            ->ifArray()
```

Comment:
just wondering: do we miss an `->end()` call? this call is indented usually also after beforeNormalization

```suggestion
                ->ifArray()
```

Reply:
The `end` call is needed only if we want to continue the chain calls, which is why it does not break here. But it would indeed be safer to add it (the old code did not have it, and did not have the extra indentation level, which is where this is coming from)

Reply:
And actually, the end call was not really missing.
The `->end()` call that was still there was going out of the root node, which does not make sense.